src/twisted/web/newsfragments/12116.feature

```diff
@@ -0,0 +1 @@
+twisted.web.http_headers now uses less CPU.
```

Comment:
Can you put a rough number on this? "Making a small HTTP request X% faster on average?"